cf-secret/cf-secret.c

```diff
@@ -126,14 +126,14 @@ static const char *const HINTS[] =
     "Specify how detailed logs should be. Possible values: 'error', 'warning', 'notice', 'info', 'verbose', 'debug'",
     "Enable basic information output",
     "Comma-separated list of key files to use (one of -k/-H options is required for encryption)",
-    "Comma-separated list of hosts to encrypt/decrypt for (defaults to 'localhost' for decryption)",
+    "Comma-separated list of hosts to encrypt/decrypt for (defaults to 'localhost')",
```

Comment:
The line above says there's no default for encryption.

Reply:
https://github.com/cfengine/core/pull/5690